Map/Source/SquareMap.cpp

```diff
@@ -38,6 +39,50 @@ char SquareMap::printField(const Field& field)
   }
 }
 
+std::string SquareMap::getMapToPrint(const Position& playerPosition)
+{
+	std::stringstream output;
+	
+	const unsigned int mapSize = _fieldList.size();
+	
+	output << "   ";
+	for(unsigned int ite = 0; ite < mapSize; ++ite)
```

Comment:
te wszystkie fory wydziel do osobnych funkcji, bedzie to duzo czytelnijesze.

Tutaj np by bylo cos w stylu ostream printColumnNubmers(conts int mapSize)

Inne fory tak samo